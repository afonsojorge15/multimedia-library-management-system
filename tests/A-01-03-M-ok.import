BOOK:Bairro Alto e o Budismo Zen:Zun Tse Fonseca:25:FICTION:1234567891:50
