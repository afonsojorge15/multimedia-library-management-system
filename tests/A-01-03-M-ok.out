Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
0 - Sair
Escolha uma opção: 0 - 50 de  50 - Livro - Bairro Alto e o Budismo Zen - 25 - Ficção - Zun Tse Fonseca - 1234567891
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 